ajax/libs/instagram-lite/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "instagram-lite",
+  "version": "2.0.0",
+  "filename": "instagramLite.min.js",
+  "description": "A simple, lightweight jQuery plugin used to display a user's Instagram photos.",
+  "keywords": [
+    "instagram",
+    "photos",
+    "api",
+    "feed"
+  ],
+  "license": "MIT",
+  "author": "Michael Lynch <mikethomaslynch@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/michael-lynch/instagram-lite"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/michael-lynch/instagram-lite.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "instragramLite*"
```

Comment:
Should be `instagramLite*` instead of `inst"r"agramLite*`. Sorry for carelessness.